src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1785,17 +1791,17 @@ private void hitBlockWithToolInHand(@Nullable final BlockPos blockPos, final boo
             if (!CompatibilityUtils.getWorld(this).isRemote)
             {
                 MineColonies.getNetwork().sendToAllAround(
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)